experiments/tool/README~

```diff
@@ -1,6 +0,0 @@
-pmd-bin-5.2.0 directory provides an instance of PMD integrated with our tool ready to be used.
```

Comment:
Please don't commit these files (*~). You can edit the .igtignore to purposefully ignore these files for you.